### DISCUSSION TITLE
How to mod among us

### DISCUSSION THREAD

**Discussion by Shaurya1901** [Woodi-dev/Among-Us-Sheriff-Mod#99] [Q&A]
Please tell me how to mod among us. I have been trying for a very long time pls help me.

**Comment by Miniflint** [comment]
2 options :
1. You have to download the zip, locate and copy your among us file, put the zip in the among us - copy, extract it

2. I made a script to download and install it. If you wanna try

**Reply by Shaurya1901** [reply]
Can you send me the script to download and install. But what will it download?
And I did not understand the first step. Which zip should I download.

**Reply by Shaurya1901** [reply]
Oh and I am using steam for among us

**Reply by Miniflint** [reply]
> Can you send me the script to download and install. But what will it download?
> 
> And I did not understand the first step. Which zip should I download. 


https://github.com/Miniflint/AmongUs-ModDownloader

**Reply by Shaurya1901** [reply]
But this is to download and install mods
I want to make mods

**Reply by Shaurya1901** [reply]
And even if I want to download mods
What should I do with the script

**Reply by Miniflint** [reply]
if you want to download mod, you just have to enter a github 
ex : https://github.com/Woodi-dev/Among-Us-Sheriff-Mod and it will download and install it automatically

if you want to MAKE mods, you have to learn C#, and how coding in general works

**Reply by Shaurya1901** [reply]
First of all, are you saying if I click on the link it will automatically download
Because it does not do that

Second I know a little bit of c#.

**Reply by Shaurya1901** [reply]
But I do not know how to make an among us mod using C#

**Reply by Miniflint** [reply]
> First of all, are you saying if I click on the link it will automatically download
> 
> Because it does not do that
> 
> 
> 
> Second I know a little bit of c#. 

I haven't said it does that
I said that if you download and open it, and inputbox will come and ask you for a download link
Once u put it, it will install the mod

**Reply by Miniflint** [reply]
> But I do not know how to make an among us mod using C#

Learn c# and then you can make mods

**Reply by Shaurya1901** [reply]
While downloading your mod downloader it shows me that it is unsafe

**Reply by Shaurya1901** [reply]
And so it does not let me open it also

**Reply by Miniflint** [reply]
> 
> 
> While downloading your mod downloader it shows me that it is unsafe

it's normal, it's an executable
you can download anyone's ".exe" file and windows will protect it
if you don't trust it, you can see the source code
and if you have any problem you can join me at : https://discord.gg/jh7PWzamNZ

**Reply by Miniflint** [reply]
windows wont let you execute exe file without any particular signature
a signature is : when you pay microsoft and they let you execute thing on windows
